extensions/markdown-mode/lem-markdown-mode.asd

```diff
@@ -1,4 +1,5 @@
-(defsystem "lem-markdown-mode"
+(asdf:defsystem "lem-markdown-mode"
```

Comment:
I'm afraid you will be tempted to add this "asdf:" to every .asd files. I learned (the hard way) that it may not be necessary (and is discouraged by the ASDF doc). We should load .asd files with `asdf:load-asd` instead of `load`, in that case load-asd knows the defsystem. In Emacs and Lem, C-c C-k in the .asd uses asdf:load-asd too. And yeah, load-asd wants a full path and doesn't work with just (asdf:load-asd "myfile.asd") but "path/to/myfile.asd", annoying…

I feel you and I'm also very tempted, my 2c.

Reply:
Okay, I think Lem might have a bug here then.

![Screenshot from 2024-02-18 06-25-39](https://github.com/lem-project/lem/assets/43155857/668b758a-1326-4722-bc53-af126f003ae1)